upgrade-2.x.sh

```diff
@@ -363,6 +363,13 @@ function post_update_jetson_fix {
             echo "extra_os_cmdline=${APPEND_value}" >> "${uEnv_file}" && sync "${uEnv_file}"
         fi
     fi
+
+    if [ -e "${uEnv_file}" ] && grep -q os_bc_lim "${uEnv_file}"; then
```

Comment:
```suggestion
    if [ -e "${uEnv_file}" ] && grep -q '^os_bc_lim=' "${uEnv_file}"; then
```
I haven't tested this, but I think we can be more specific with our grep here.

Reply:
Sure, done @klutchell